packages/dds/sequence/src/test/sharedString.spec.ts

```diff
@@ -249,21 +250,14 @@ describe("SharedString", () => {
 			assert.equal(simpleMarker.properties?.color, "blue", "Could not annotate marker");
 			// Annotate the marker's ID.
 			const newIdProps = { [reservedMarkerIdKey]: "newIdValue" };
-			try {
-				sharedString.annotateMarker(simpleMarker, newIdProps);
-				assert.fail();
-			} catch (error) {
-				assert.notEqual(
-					error,
-					undefined,
-					"Error from attempting to update marker was not thrown",
-				);
-				assert.equal(
-					(error as Error).message,
-					"Cannot change the markerId of an existing marker",
-					`Incorrect error received - got ${(error as Error).message}`,
-				);
-			}
+			assert.throws(
+				() => {
+					sharedString.annotateMarker(simpleMarker, newIdProps);
+					assert.fail("Error from attempting to update marker was not thrown");
```

Comment:
Last nit from me 😄 . If we use `assert.throws()` then I think we can skip the `assert.fail()` (`assert.throws()` will already fail the test if the code block doesn't throw, which handles @connorskees 's previous concern), and move this message to the be last parameter to `assert.throws()`, so it prints it as the result of "what I expected to happen (an error that meets the validation function) didn't happen". And if we do, the message probably need an update to `"Error from attempting to update marker was not thrown or was not the expected error"`.